recipe/conda_build_config.yaml

```diff
@@ -167,7 +173,7 @@ pixman:
 poppler:
   - 0.61
 proj4:
-  - 4
+  - 4.9.3
 protobuf:
   - 3.4
```

Comment:
We should bump this to `3.5`. Also planning on making this `libprotobuf` as that will be the C++ only package. Will need to keep it at `3.5` for VS 2008 support as it is the last non-C++11 release, but will be a LTS release.

Reply:
Opened as issue ( https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/14 ).